velox/docs/monthly-updates/may-2024.rst

```diff
@@ -0,0 +1,69 @@
+***************
+May 2024 Update
+***************
+
+Documentation
+=============
+
+* Publish blog post about `optimizing TRY and TRY_CAST <https://velox-lib.io/blog/optimize-try-more>`_
```

Comment:
We can also add the build metrics page here : https://facebookincubator.github.io/velox/bm-report/

Reply:
Thank you. Added.